src/Nav/README.md

```diff
@@ -58,3 +58,9 @@ Navigation bits in Bootstrap all share a general ``Nav`` component and styles. S
   );
 }
 ```
+
+<guide
+  events="`onKeyDown`, `onClick`"
+  dataTestId
+  selectors="`nav`, `nav nav-pills`"
```

Comment:
looks like `nav` is included twice here

Reply:
yes, can be removed, probably it was meant to indicate that it is the same element but different variants